sapientml_core/templates/model_templates/model.py.jinja

```diff
@@ -45,4 +58,4 @@ y_pred = model.classes_[np.argmax(y_pred, axis=1)].reshape(-1, 1)
 {% endif %}
 {% if model_name == xgbclassifier and (not pipeline.adaptation_metric.startswith("MAP_")) and (not pipeline.adaptation_metric == "LogLoss") and (pipeline.adaptation_metric not in metric_needing_predict_proba) %}
 y_pred = label_encoder.inverse_transform(y_pred).reshape(-1, 1)
```

Comment:
Could you also implement inverse operation for the multicolumn targets? You would need to use `label_encoders` defined above.